README.md

```diff
@@ -1,3 +1,11 @@
-Take this course if you'd like to learn how to write and publish your own course on GitHub Learning Lab.
+Author your own course on GitHub Learning Lab in just a few minutes! 
 
-You'll need to be a owner of this organization in order to publish the course you create.
\ No newline at end of file
+After completing this course, you'll be able to:
+- identify the components of a course on GitHub Learning Lab
+- create course metadata
```

Comment:
```suggestion
- write metadata for machine readability 
```
^ I'm not sure this suggestion is a great solution, but do you think the user needs any extra understanding for this step? I could be overthinking it.